lookout/style/typos/templates/comment.md.jinja2

```diff
@@ -0,0 +1,8 @@
+format: typos:
```

Comment:
Why is is exactly `format: typos:`? Didn't you mean `style: typos:`? Won't there be any confusion between `format` and `typos` analyzers?

Reply:
yep, WDYT about `typos: misspelled identifiers correction:`?